pages/manual/planet/overview.md

```diff
@@ -52,14 +52,13 @@ When you are inside PLANET you will see different features available.
 List of features:
 1. Library
 2. Courses
-3. Exams
```

Comment:
Lets keep Exam section here. If content is not there you can put work in progress or something similar. Removing it will make it not worked at all.